book/src/project-policies.md

```diff
@@ -22,6 +22,14 @@ A litmus test for not breaking compatibility is to replace the image of a
 component in an existing deployment without changing that deployment in any
 other way.
 
+To minimize the number of branches we need to support, we do not have a general
+policy for releasing new minor versions on older majors. We will make exceptions
+for work related to meeting production readiness requirements. Only the previous
+major version will be eligible for these exceptions, so long as the time between
+the previous major version and the current major version is under four months.
```

Comment:
Thinking about this some more, given that the Kubernetes release cadence is slowling down, I wonder if 4 months is actually enough enough to capture even a single release. Should we bump it to 6 months? @xing-yang

Reply:
Sounds good.

Reply:
Done